src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -248,6 +246,10 @@ AndroidDeviceControllerWrapper * AndroidDeviceControllerWrapper::AllocateNew(
     }
     else
     {
+        ChipLogProgress(Controller,
+                        "No existing credentials provided: generating ephemeral local NOC chain with OperationalCredentialsIssuer",
+                        static_cast<unsigned>(wrapper->Controller()->GetFabricIndex()));
```

Comment:
@tcarmelveilleux  -  compiler complains about unused format argument here

Reply:
Will fix. Some issues arose as I adapted internal code for upstreaming